packages/framework/src/validators/validator.test.ts

```diff
@@ -289,18 +289,22 @@ describe('validators', () => {
               {
                 message: "must have required property 'stringVal'",
                 path: '',
+                property: '.stringVal',
               },
               {
                 message: 'must be number',
                 path: '/numVal',
+                property: '.numVal',
               },
               {
                 message: "must have required property 'boolVal'",
                 path: '',
+                property: '.boolVal',
               },
               {
                 message: 'must match a schema in anyOf',
                 path: '',
+                property: '',
               },
             ],
           },
```

Comment:
Can we add a new test for validating an array to demonstrate how a `path` is formed for a invalid property inside array data?